### PR TITLE
Issue #357: Informative error message on connection failure

### LINKED ISSUE #357: User friendly self-signed certificate warning message
The most common error encountered on the console is when login fails due to a self-signed cert. However the error message is too generic and confusing for new users. If possible, the console should detect this error and provide an appropriate message. Perhaps instructions could also be displayed helping the user resolve the problem (with a link to the api where they need to accept the cert from)

### PR DESCRIPTION
**Pull request by hnnesv**
Closes #357 

@rcrichton @BMartinos I thought it'd be great to get this in before the next release.

Unfortunately I can't see a good way to specifically determine that it's a cert issue (and seems to be a general [problem](http://stackoverflow.com/questions/31058764/determine-if-ajax-call-failed-due-to-insecure-response-or-connection-refused)) so just wrote it up generically

### CODE REVIEW

**Comment by hnnesv** [comment]
screen shot 2016-02-15 at 10 37 25

**Comment by BMartinos** [comment]
:+1: looks good to me

File: app/views/login.html [hunk 1]
```diff
@@ -45,8 +45,15 @@
         </div>
       </div>
 
+      <div ng-if="coreConnectionError" class="alert alert-danger" style="padding: 10px; margin: 15px 10%;">
+          An error occurred while attempting to connect to the OpenHIM Core.<br><br>
```
Comment (rcrichton) [inline app/views/login.html]: Could we perhaps list the connection URL that the API attempted to connect to?

**Comment by rcrichton** [comment]
Looks good, just a small comment above.

**Comment by hnnesv** [comment]
@rcrichton ok cool: 
screen shot 2016-02-15 at 12 45 08

**Comment by rcrichton** [comment]
Cool, looks good to me :+1:

**Pull request merged**